sigma/pipelines/netwitness/fortinet.py

```diff
@@ -0,0 +1,116 @@
+"""Module for pySigma NetWitness processing pipelines"""
+
+from typing import Dict, List, Tuple, Union
+
+from sigma.processing.conditions import LogsourceCondition
+from sigma.processing.pipeline import ProcessingItem, ProcessingPipeline
+from sigma.processing.transformations import AddConditionTransformation, FieldMappingTransformation
+
+netwitness_fortinet_field_mappings: Dict[str, Union[str, List[str]]] = {
```

Comment:
Are these predefined values from NetWitness or are those customizable. If they are not predefined we should consider outsourcing them into some kind of configuration.

Reply:
These are customizable. I would like to keep the mappings inside python modules so that I don't have IO overhead.

Reply:
But wouldn't this mean anyone using the module would have to fork it to customize this?